channeldb/graph.go

```diff
@@ -2855,7 +2914,7 @@ func fetchChanEdgePolicies(edgeIndex *bolt.Bucket, edges *bolt.Bucket,
 	// something other than edge non-existence.
 	node1Pub := edgeInfo[:33]
 	edge1, err := fetchChanEdgePolicy(edges, chanID, node1Pub, nodes)
-	if err != nil && err != ErrEdgeNotFound {
+	if err != nil {
```

Comment:
hmm.. rationale for this change?

Reply:
For `chanID` we now always expect to find two policies (that may be unknown). Previously, `ErrEdgeNotFound` was happy flow.